src/both/DeltaNeutralLpV3.sol

```diff
@@ -471,15 +514,27 @@ contract DeltaNeutralLpV3 is AccessStrategy {
     /// @dev Gives ratio of vault asset to borrow asset, e.g. WMATIC/USD (we assume that usd = usdc)
     AggregatorV3Interface immutable borrowFeed;
 
-    /// @dev Convert `borrow` (e.g. MATIC) to `asset` (e.g. USDC)
-    function _borrowToAsset(uint256 amountB, uint256 clPrice) internal pure returns (uint256 assets) {
-        // The first division gets rid of the decimals of wmatic. The second converts dollars to usdc
-        assets = amountB.mulWadDown(clPrice) / 1e2;
+    /**
+     * @dev Convert `borrow` (e.g. MATIC) to `asset` (e.g. USDC)
+     * 10^borrow_decimals in `borrow` = clPrice / 10^borrowFeed_decimals * 10^asset_decimals in `assets`
```

Comment:
This seems to be backwards. The clPrice is the asset/borrow ratio. This comment implies that

```
1 borrow / 1 asset = clPrice / borrowFeed_decimals
```

I think you can put some comments inside the function (not natspec) explaining that when we multiple `borrow` amounts by the `clPrice` we have a number with `borrowDecimals + borrowFeedDecimals`. We optionally add or remove decimals based on the value of `assetDecimals`